src/test/resources/crashes/minecraft/modded-level-file-parsed.txt

```diff
@@ -1,8 +1,9 @@
world.level.saveddata.maps.MapItemSavedData)(Level.java:564)\n\tat net.minecraft.world.level.Level.void destroyBlockProgress(int,net.minecraft.core.BlockPos,int)(Level.java:564)\n\tat net.minecraft.server.level.ServerLevel.void tick(java.util.function.BooleanSupplier)(ServerLevel.java:411)\n\tat net.minecraft.server.MinecraftServer.void tickChildren(java.util.function.BooleanSupplier)(MinecraftServer.java:871)\n\tat net.minecraft.server.dedicated.DedicatedServer.void tickChildren(java.util.function.BooleanSupplier)(DedicatedServer.java:312)\n\tat net.minecraft.server.MinecraftServer.void tickServer(java.util.function.BooleanSupplier)(MinecraftServer.java:811)\n\tat net.minecraft.server.MinecraftServer.boolean publishServer(net.minecraft.world.level.GameType,boolean,int)(MinecraftServer.java:811)\n\tat net.minecraft.server.MinecraftServer.boolean isSingleplayerOwner(com.mojang.authlib.GameProfile)(MinecraftServer.java:811)\n\tat net.minecraft.server.MinecraftServer.void runServer()(MinecraftServer.java:670)\n\tat net.minecraft.server.MinecraftServer.net.minecraft.server.MinecraftServer spin(java.util.function.Function)(MinecraftServer.java:257)\n\tat net.minecraft.server.MinecraftServer.boolean publishServer(net.minecraft.world.level.GameType,boolean,int)(MinecraftServer.java:257)\n\tat net.minecraft.server.MinecraftServer.boolean isSingleplayerOwner(com.mojang.authlib.GameProfile)(MinecraftServer.java:257)\n\tat net.minecraft.server.MinecraftServer.void lambda$spin$0(java.util.concurrent.atomic.AtomicReference)(MinecraftServer.java:257)\n\tat java.base/java.lang.Thread.run(Thread.java:834)\n\n-- Affected level --\nDetails:\n\tAll players: 1 total; [aah['Marcono1234'/292, l='ServerLevel[world]', x=-659.01, y=95.90, z=99029.02]]\n\tChunk stats: ServerChunkCache: 3079\n\tLevel dimension: minecraft:overworld\n\tLevel spawn location: World: (-208,67,208), Chunk: (at 0,4,0 in -13,13; contains blocks -208,0,208 to -193,255,223), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)\n\tLevel time: 17947 game time, 17947 day time\n\tLevel name: world\n\tLevel game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false\n\tLevel weather: Rain time: 140516 (now: false), thunder time: 23783 (now: false)\n\tKnown server brands: vanilla\n\tLevel was modded: true\n\tLevel storage version: 0x04ABD - Anvil\n\n-- System Details --\nDetails:\n\tMinecraft Version: 1.16.5\n\tMinecraft Version ID: 1.16.5\n\tOperating System: Windows 10 (amd64) version 10.0\n\tJava Version: 11.0.5, AdoptOpenJDK\n\tJava VM Version: OpenJDK 64-Bit Server VM (mixed mode), AdoptOpenJDK\n\tMemory: 425461760 bytes (405 MB) / 1073741824 bytes (1024 MB) up to 2147483648 bytes (2048 MB)\n\tCPUs: 6\n\tJVM Flags: 0 total; \n\tPlayer Count: 1 / 20; [aah['Marcono1234'/292, l='ServerLevel[world]', x=-659.01, y=95.90, z=99029.02]]\n\tData Packs: vanilla, file/MC-224778.zip (incompatible)\n\tIs Modded: Unknown (can't tell)\n\tType: Dedicated Server (map_server.txt)\n",
```

Comment:
This change comes from the fact that the crash report is now not considered modded anymore and is therefore deobfuscated.